lib/check-self-update/request-promise.js

```diff
@@ -0,0 +1,14 @@
+const request = require('request');
```

Comment:
@rafaelcalpena The `request` library is quite large for this singular use case. To keep the `lsc` project dependencies smaller, can you use a less extensive request library instead? Perhaps https://www.npmjs.com/package/tiny-json-http.